bindings/nodejs/examples/how_tos/accounts_and_addresses/check_balance.ts

```diff
@@ -2,24 +2,28 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Wallet, initLogger } from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
 require('dotenv').config({ path: '.env' });
 
 // Run with command:
-// yarn run-example ./wallet/02_get_balance.ts
+// yarn run-example ./how_tos/accounts_and_addresses/check_balance.ts
 
 // This example syncs the account and prints the balance
 async function run() {
     initLogger();
     try {
         const wallet = new Wallet({
-            storagePath: './alice-database',
+            storagePath: process.env.WALLET_DB_PATH,
         });
 
-        const account = await wallet.getAccount('Alice');
+        const account = await wallet.getAccount(
+            `${process.env.ACCOUNT_ALIAS_1}`,
```

Comment:
No need to "format" it
```suggestion
            process.env.ACCOUNT_ALIAS_1,
```

Reply:
Cant, its string |undefined instead of string. and the ! will be illegal :(

Reply:
Scam